.github/workflows/rust.yml

```diff
@@ -1,5 +1,5 @@
-# Rust specific tests like formatting, clippy, etc
-name: Rust checks
+# Rust specific tests
+name: Rust
```

Comment:
If you worry that this could be confused with actual tests written in rust, we could also go with something like "Rust source code".

Reply:
I've stopped worrying a long time ago :)